pkg/net/http/httputils.go

```diff
@@ -40,26 +42,24 @@ type Pagination struct {
 	SortOrder string
 	StartDate time.Time
 	EndDate   time.Time
+	Alias     string
 }
 
 // ValidateParameters validate and return struct of default parameters
 func ValidateParameters(params map[string]string) (*QueryHeader, error) {
-	var metadata *bson.M
-
-	var portfolioID string
-
-	var toAssetCodes []string
-
-	var startDate time.Time
-
-	var endDate time.Time
-
-	var cursor string
-
-	limit := 10
-	page := 1
-	sortOrder := "desc"
-	useMetadata := false
+	var (
+		metadata     *bson.M
+		portfolioID  string
+		toAssetCodes []string
+		startDate    time.Time
+		endDate      time.Time
+		cursor       string
+		alias        string
```

Comment:
Validate if alias should be +1

```suggestion
	Aliases        []string
```